client/src/ducks/post.js

```diff
@@ -103,21 +105,12 @@ const INITIAL_STATE = {
     error_message: "",
     showPassword: false,
     password: '',
-    lost_locations: [
-        ["Tech", 25],
-        ["Norris", 15],
-        ["Plex", 4],
-        ["Sheridan Rd", 1],
-        ["Annenberg", 1],
-        ["SPAC", 1]
-    ],
-    found_locations: [
-        ["Norris", 10],
-        ["Plex", 4],
-        ["Sheridan Rd", 1],
-        ["SPAC", 1]
-    ],
-    location_search: ""
+    lost_locations: [["Tech", 25], ["Norris", 15], ["Plex", 4], ["Sheridan Rd", 1], ["Annenberg", 1], ["SPAC", 1]],
+    found_locations: [["Norris", 10], ["Plex", 4], ["Sheridan Rd", 1], ["SPAC", 1]],
+    location_search: "",
+    email_message: "",
+    responder_name : "",
+    responder_email : ""
```

Comment:
I don't see anywhere in Redux, where these 3 states are changed (email_message, responder_name, and responder_email).